src/main/java/edu/kit/provideq/toolbox/meta/ProblemRegistration.java

```diff
@@ -0,0 +1,79 @@
+package edu.kit.provideq.toolbox.meta;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * This helper class can register problems deeply in the corresponding
+ * {@link ProblemManager problem managers}.
+ */
+@Component
+public class ProblemRegistration {
```

Comment:
Do we need this class? It doesn't seem to be used and `ProblemManager` seems to have the same functionality.

Reply:
Whoops, that used to be different. You're right and I have just removed this class